test/cpp/qps/driver.cc

```diff
@@ -89,6 +90,8 @@ ScenarioResult RunScenario(const ClientConfig& initial_client_config,
     return &contexts.back();
   };
 
+  srand(time(NULL) ^ getpid());
```

Comment:
Doesn't the grpc_test_init() take care of that already ? (or gpr_test_init, can't remember). It'd be a shame to prevent Windows compilation.